Buildings/Fluid/Chillers/Absorption_Indirect_HotWater.mo

```diff
@@ -0,0 +1,272 @@
+            nPorts=2,final prescribedHeatFlowRate=true),
+      vol3(energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+           massDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+           V=m3_flow_nominal*tau3/rho3_nominal,
+           nPorts=2,final prescribedHeatFlowRate=true));
+
+  parameter Buildings.Fluid.Chillers.Data.AbsorptionIndirect.Generic  per
+   "Performance data"
+    annotation (choicesAllMatching= true,
+       Placement(transformation(extent={{48,66},{68,86}})));
+  final parameter Modelica.SIunits.HeatFlowRate
+     QEva_heatflow_nominal= per.QEva_flow_nominal
+    "Nominal heat flow at the evaporator"
+      annotation (Dialog(group="Nominal condition"));
+  final parameter Modelica.SIunits.MassFlowRate mCon_flow_nominal= per.mCon_flow_nominal
+   "Nominal mass flow at Condenser"
+    annotation (Dialog(group="Nominal condition"));
+  final parameter Modelica.SIunits.MassFlowRate mEva_flow_nominal= per.mEva_flow_nominal
+   "Nominal mass flow at Evaorator"
+    annotation (Dialog(group="Nominal condition"));
+  final parameter Modelica.SIunits.HeatFlowRate Q_flow_small = -QEva_heatflow_nominal*1E-9
+    "Small value for heat flow rate or power, used to avoid division by zero";
+
+  /*
+  hot water flow rate
+  parameter Modelica.SIunits.SpecificHeatCapacity cpWat=
+    Medium1.specificHeatCapacityCp(Medium.setState_pTX(
+      p = Medium1.p,
+      T = Medium1.T,
+      X = Medium1.X));
+  */
+
+
+
+  BaseClasses.AbsorptionIndirect absBlo(per=per)
+   "Absorption indirect chiller equations block"
+    annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
+  Modelica.Blocks.Interfaces.BooleanInput on
+   "Chiller turn On/off inout signal "
+    annotation (Placement(transformation(extent=
+           {{-168,-14},{-140,14}}), iconTransformation(extent={{-122,-10},{-100,12}})));
+  Modelica.Blocks.Interfaces.RealInput TEvaSet(final unit="K", displayUnit="degC")
+   "Evaporator setpoint leaving water temperature"
+    annotation (Placement(
+        transformation(extent={{-166,-92},{-140,-66}}), iconTransformation(
+          extent={{-122,-100},{-100,-78}})));
+  Modelica.Blocks.Sources.RealExpression QEvaFloSet(final y=
+        Buildings.Utilities.Math.Functions.smoothMin(
+            x1 = mEvaFlo.y*(hEvaSet.y - inStream(port_a2.h_outflow)),
+            x2 = -Q_flow_small,
+        deltaX = Q_flow_small/100))
+   "Setpoint heat flow rate of the evaporator"
+    annotation (Placement(transformation(extent={{-138,-30},{-118,-10}})));
+  Modelica.Blocks.Sources.RealExpression hEvaSet(final y=
+        Medium2.specificEnthalpy_pTX(
+        p = port_b2.p,
+        T = TEvaSet,
+        X = cat(1,
+              port_b2.Xi_outflow,{1 - sum(port_b2.Xi_outflow)})))
+   "Chilled water setpoint enthalpy"
+    annotation (Placement(transformation(extent={{-138,-64},{-118,-44}})));
+  Modelica.Blocks.Sources.RealExpression mEvaFlo(y=vol2.ports[1].m_flow)
```

Comment:
Can you please add `unit` and for temperatures `displayUnit` to the `RealOutput` and `RealInput`.